### PR TITLE
NETOBSERV-1850 fix cli enrichment issue

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Fix CLI packet enrichment issue where all packets were pointing the same info

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [x] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [x] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: cmd/packet_capture.go [hunk 1]
```diff
@@ -146,8 +140,8 @@ func runPacketCaptureOnAddr(port int, filename string) {
 			// write enriched data as interface
 			writeEnrichedData(pw, &genericMap)
 
-			// then append packet to file
-			err = pw.WriteEnhancedPacketBlock(0, ts, b, types.EnhancedPacketOptions{})
+			// then append packet to file using totalPackets as unique id
+			err = pw.WriteEnhancedPacketBlock(totalPackets, ts, b, types.EnhancedPacketOptions{})
```
Comment (msherif1234) [inline cmd/packet_capture.go:144]: @jpinsonneau how the total packet can be unique id ?

Comment (jpinsonneau) [reply cmd/packet_capture.go:144]: I increase the `totalPackets` value each time we add a packet just below this call:
```
		totalPackets = totalPackets + 1
```

Comment (msherif1234) [reply cmd/packet_capture.go:144]: yeah I saw that isn't that will be the case with other pkts u capture so if I received 2 udp pkts then 2 tcp pkts in two different runs won't that collide ?

Comment (jpinsonneau) [reply cmd/packet_capture.go:144]: do you open the 2 pcapng files in one ? :thinking:

Comment (jpinsonneau) [reply cmd/packet_capture.go:144]: I tried the following code to see what happens:
```
err = pw.WriteEnhancedPacketBlock(uint32(startupTime.UnixMilli())+totalPackets, ts, b, types.EnhancedPacketOptions{})
```

And it seems wireshark is not expecting that. Here the id is an index related to the previously created interface description using `WriteInterfaceDescription` call:
image

Comment (msherif1234) [reply cmd/packet_capture.go:144]: ok I guess lets proceed as is thanks for confirming

**Comment by jotak** [comment]
To be tested post merge

**Pull request merged**